src/app/qgsdxfexportdialog.cpp

```diff
@@ -149,7 +149,7 @@ QgsVectorLayerAndAttributeModel::QgsVectorLayerAndAttributeModel( QgsLayerTree *
     const QgsVectorLayer *vLayer = qobject_cast< const QgsVectorLayer *>( QgsProject::instance()->mapLayer( id ) );
     if ( vLayer )
     {
-      mCreateDDBlockInfo[vLayer] = QgsDxfExportDialog::settingsDxfEnableDDBlocks->value();
+      mCreateDDBlockInfo[vLayer] = true;
```

Comment:
It would be really handy to have a constant that can be used across all DXF Export classes to access this default value. 

Just saying because I'm preparing another PR (see [my local PR](https://github.com/gacarrillor/QGIS/pull/38) in the meantime), where I actually need the default value to improve code readability and maintenance (namely [here](https://github.com/gacarrillor/QGIS/pull/38/files#diff-9c2a19b9e6181feed587a5a1527cf7541cc75a42196aeffab160bacd249417e9R600) to have something like `if ( allowDataDefinedBlocks != DEFAULT_DXF_DATA_DEFINED_BLOCKS)`).

Without such constant, it would be too easy for someone to break something in the future regarding this default value.

I guess it could live in the `dxfexport.h` or in the [qgis.cpp](https://github.com/qgis/QGIS/blob/810a61a791e92af1bbfdb2483764228c9f140ade/src/core/qgis.cpp#L40-L70). 

Would that make sense for you @mhugent?

Reply:
Yes, sounds good to have it in dxfexport.h